get.sh

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env bash
+
+: '
+    Copyright (C) 2020 IBM Corporation
+
+    Rafael Sene <rpsene@br.ibm.com> - Initial implementation.
+
+    This is a helper script that eases the dependency setup and 
+    configuration for the pvsadm tool. 
+'
+
+# Trap ctrl-c and call ctrl_c()
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "Bye!"
+}
+
+function identify_os() {
+
+    local OS="$(uname -s)"
+
+    case "${OS}" in
+        Linux*)     DISTRO=linux;;
+        Darwin*)    DISTRO=darwin;;
+        Catalina*)  DISTRO=darwin;;
+        *)          DISTRO="UNKNOWN:${OS}"
+    esac
+
+    ARCH=$(uname -m)
+
+    if [ "$ARCH" == "amd64" ] || [ "$ARCH" == "x86_64" ]; then
+        ARCH=amd64
+    fi
+
+    export $ARCH
+    export $DISTRO
+}
+
+function check_connectivity() {
+    
+    curl --output /dev/null --silent --head --fail http://ibm.com
+    if [ ! $? -eq 0 ]; then
+        echo
+        echo "ERROR: please, check your internet connection."
+        exit
+    fi
+}
+
+function install_pvsadmin() {
+
+    PVSADM_VERSION=$1
+
+    if [ -z "$PVSADM_VERSION" ]; then
+        echo
+        echo "ERROR: pvsadm version was not set."
+        echo "       select a version from the list below."
+        echo "       get.sh <VERSION>"
+        echo
+
+        curl -s "https://api.github.com/repos/ppc64le-cloud/pvsadm/releases" >> /tmp/pvsadmin.json
+
+        TAGS=($(cat /tmp/pvsadmin.json | jq -r '.[].tag_name'))
+        for tag in "${TAGS[@]}"; do
+            echo $tag
+        done
+        rm -f /tmp/pvsadmin.json
+        exit
+    fi
+
+    rm -f /tmp/pvsadmin.json
+
+    if command -v "pvsadm" &> /dev/null; then
+        echo "pvsadm is already installed!"
+        exit
+    fi
+
+    curl --progress-bar -LJ "https://github.com/ppc64le-cloud/pvsadm/releases/download/$PVSADM_VERSION/pvsadm-$DISTRO-$ARCH" --output /usr/local/bin/pvsadm
+
+    chmod +x /usr/local/bin/pvsadm
+    pvsadm version
+}
+
+function run (){
+
+    identify_os
+    check_connectivity
+    install_pvsadmin $1
+}
+
+run "$@"
```

Comment:
a new line can be added at the end of the file

Reply:
This can also install the qemu-img and growpart dependencies that are needed for image conversion on ppc64le.

Reply:
We can just name this script `install` and add a simple description in README to `curl` the file and run it for install. Something like
`curl -sL <path-to-install> | bash`

Reply:
I prefer to keep this script as simple as possible to use this script in a generic way even for downloading on the mac just for import/upload for which you don't need to worry about those 2 packages. Having that said, if you want to have some flag to install them and not by default - I'm okay.

Reply:
one more comment is: to make this above curl command work with the script, we need think about feeding the version via environment instead of command-line argument.